.github/workflows/nix.yml

```diff
@@ -21,3 +23,5 @@ jobs:
         uses: actions/checkout@v2
       - name: Build
         run: nix build
+      - name: Check
+        run: nix flake check
```

Comment:
This wont work because of a bug in how IFD is handled in these properties right now.

see https://github.com/NixOS/nix/issues/4265

Reply:
Oh, you were quicker than me, I was just writing a comment asking you whether that's expected or not. I'll drop it for now then, looks like you do know what you're doing. Good! :-)

Reply:
Yeah I would have added the `nix check` myself if not for this issue.

I think there are plans to allow nix check of specific fields, so we can at least do that if the main issue is not fixed when that happens.